_posts/2024-01-29-advanced-authorization.adoc

```diff
@@ -0,0 +1,95 @@
+---
+title: Advanced Authorization
+layout: post
+author: Ilya Buziuk
+description: >-
+  The article describes how to configure advanced authorization in Eclipse Che
+categories: []
+keywords: ['postgresql']
+slug: /@ilya.buziuk/advanced-authorization
+---
+
+Even though the current trend is instead of having a gigantic monolith {kubernetes} cluster split the infrastructure into a number of 'Fit For Purpose' clusters, administrators still want to provide granular access and restrict certain functionality for particular users.
+
+NOTE: 'Fit For Purpose' {kubernetes} cluster refers to a cluster specifically designed and configured to meet the requirements and goals of a particular use case or workload, tailored to optimize performance, resource utilization, and other factors based on the characteristics of the workloads it will be managing. For {prod} it is recommended to have this type of cluster provisioned.
```

Comment:
```suggestion
NOTE: A 'Fit For Purpose' {kubernetes} cluster refers to a cluster specifically designed and configured to meet the requirements and goals of a particular use case or workload, tailored to optimize performance, resource utilization, and other factors based on the characteristics of the workloads it will be managing. For {prod} it is recommended to have this type of cluster provisioned.
```